src/components/SightingDetail/SightingDetail.tsx

```diff
@@ -0,0 +1,38 @@
+import "./SightingDetail.scss"
+import React, { useEffect, useState } from 'react';
+import { fetchSightingById, WhaleSighting} from "../../clients/apiClient";
+import "./SightingDetail.scss";
+
+interface SightingDetailProps {
+    sightingId: number;
+}
+export function SightingDetail(props: SightingDetailProps): JSX.Element {
+    const [sighting, setSighting] = useState<any>(null);
+
+    useEffect(() => {
+        fetchSightingById(props.sightingId)   
+        .then(response => { setSighting(response); });
```

Comment:
As this method is on a single line do we need the curly brackets?
```suggestion
        .then(response => setSighting(response));
```